Packs/Cymptom/README.md

```diff
@@ -0,0 +1,12 @@
+### What does this pack do?
+
+By leveraging the integration between Cortex® XSOAR and Cymptom’s security posture management solution, security teams can automate and optimize complex workflows across the full stack of information systems and security tools. 
+Together, Cortex XSOAR and Cymptom’s integrated solutions provide always-on risk visibility that security teams need throughout their on-premise and cloud-based networks. 
+By leveraging this integration, your team optimizes existing security tools and workflows with full visibility of vulnerabilities and prioritized risk. 
+- Integrate always-on risk & vulnerability assessment with the rest of your security stack and workflows for more comprehensive visibility and a continuous risk remediation.  

```

Comment:
This sentence is a little confusing. Would be better if we can change it to:
"Integrate with Cymptom's always-on comprehensive risk & vulnerability assessment with the rest of your security stack and workflows for more comprehensive visibility and a continuous risk remediation.